src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -316,7 +357,10 @@ class WorkspaceMembersPage extends React.Component {
                             title={this.props.translate('workspace.common.members')}
                             subtitle={policyName}
                             onCloseButtonPress={() => Navigation.dismissModal()}
-                            onBackButtonPress={() => Navigation.navigate(ROUTES.getWorkspaceInitialRoute(policyID))}
+                            onBackButtonPress={() => {
+                                this.clear();
```

Comment:
Why is this necessary?

Reply:
@0xmiroslav @marcaaron  we can keep this?

Reply:
I'm not following why we need to do anything or what problem it solves.